bookmarks/templates/bookmarks/bookmarklet.js

```diff
@@ -1,8 +1,10 @@
 (function () {
   var bookmarkUrl = window.location;
+  var bookmarkTitle = document.title;
```

Comment:
As mentioned in the linked issue I'm not convinced that client-side scraping is a better solution, there are a lot of SPA type applications that fail to properly update the document title / description metadata on navigation. For now I would only offer the possibility to initialize the title + description in the form, then people can at least use a custom bookmarklet if they prefer client-side scraping.

Reply:
I have reverted the bookmarklet change.